k8s/helmfile/env/common/elasticsearch.values.yaml.gotmpl

```diff
@@ -0,0 +1,23 @@
+image: "wikibase/elasticsearch"
+imageTag: "6.8.23-wmde.6"
```

Comment:
Should we repeat the version number for each env so we can have an easier rolling upgrade?

Reply:
yes please :)

Reply:
Done so.